pkgs/tools/misc/barman/default.nix

```diff
@@ -34,6 +34,8 @@ python3Packages.buildPythonApplication rec {
   ];
 
   disabledTests = [
+    # FsOperationFailed
+    "test_get_file_mode"
```

Comment:
Only on Darwin? If so, only disable the check on Darwin.

Reply:
Done.